docs/content/1.getting-started/3.configuration.md

```diff
@@ -35,3 +35,21 @@ export default defineNuxtConfig({
 ::callout
 As result show 20/12/2020 instead of 20 December 2020
 ::
+
+## `codesMap`
```

Comment:
@zAlweNy26  Thanks for you're help.

I think for better naming `codesMap` should be rename to `localeCodesMapping`

Reply:
Yeah, you are right, I'll fix it